packages/xdl/src/AssetUtils.ts

```diff
@@ -33,14 +33,24 @@ export const optimizeImageAsync = async (inputPath: string, quality: number) =>
     quality,
   });
   return path.join(outputPath, path.basename(inputPath));
+}
+
+export type OptimizationOptions = {
+  quality: number;
+  include?: string;
+  exclude?: string;
+  save?: boolean;
 };
 
-type Options = { quality: number; include: string; exclude: string };
+export type AssetInfo = { [hash: string]: boolean };
```

Comment:
I'm thinking it might be better to call this `ProjectAssetInfo` or something that explains that this contains information about the assets in the project because `AssetInfo` could also describe the info for a single asset (like suggested in one of my previous comments :P)

Reply:
Renamed to `AssetOptimizationState`.